schulcloud/UserPermissions.swift

```diff
@@ -16,6 +16,12 @@ struct UserPermissions : OptionSet {
         self.rawValue = rawValue
     }
     
+    init(array: [String]) {
```

Comment:
This can be marked as `convenience`

Reply:
Marking an `init()` as `convenience` in a `struct` is not legal Swift unfortunately.